### PR TITLE
flow: introduce flow stream endpoint

### PR DESCRIPTION
**Pull request by safchain**
This aims to provide a way to get flow
in through websocket endpoint. It supports
flow as JSON or Protobuf according to the given
protocol of the client.

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-ppc64le-tests

**Comment by safchain** [comment]
run  skydive-ppc64le-tests

**Comment by safchain** [comment]
run  skydive-ppc64le-tests

**Comment by safchain** [comment]
run  skydive-ppc64le-tests

**Comment by safchain** [comment]
run  skydive-ppc64le-tests

**Comment by safchain** [comment]
run  skydive-ppc64le-tests

File: analyzer/flow_subscriber_endpoint.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	"github.com/skydive-project/skydive/flow"
+	ws "github.com/skydive-project/skydive/websocket"
+)
+
+// FlowSubscriberEndpoint sends all the flows to its subscribers.
+type FlowSubscriberEndpoint struct {
+	common.RWMutex
+	ws.DefaultSpeakerEventHandler
+	pool ws.StructSpeakerPool
+	wg   sync.WaitGroup
```
Comment (lebauce) [inline analyzer/flow_subscriber_endpoint.go]: Seems to be unused

File: analyzer/flow_subscriber_endpoint.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	"sync"
+
+	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/flow"
+	ws "github.com/skydive-project/skydive/websocket"
+)
+
+// FlowSubscriberEndpoint sends all the flows to its subscribers.
+type FlowSubscriberEndpoint struct {
+	common.RWMutex
```
Comment (lebauce) [inline analyzer/flow_subscriber_endpoint.go]: Seems to be unused

**Comment by safchain** [comment]
run  skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run  skydive-cdd-overview-tests

**Comment by safchain** [comment]
run tests

**Comment by safchain** [comment]
rerun tests

**Pull request merged**